source/github-events/on-comment-field-edit.tsx

```diff
@@ -0,0 +1,18 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+
+export default function listenToCommentFields(callback: delegate.EventHandler<KeyboardEvent, HTMLTextAreaElement>): void {
+	delegate<HTMLTextAreaElement, KeyboardEvent>(document, '.js-comment-field, #commit-description-textarea', 'keydown', event => {
+		const field = event.delegateTarget;
+
+		// Don't do anything if the autocomplete helper is shown or if non-Roman input is being used
```

Comment:
By **_"non-roman"_** don't you mean **_"non-latin"_**? Or am I missing something?

Reply:
[Good catch!](https://en.wikipedia.org/wiki/Keyboard_layout)